src/Compiler/TypedTree/TcGlobals.fs

```diff
@@ -911,15 +904,9 @@ type TcGlobals(
             ILType.Value (mkILNonGenericTySpec tref)
         mkILCustomAttribute (findSysILTypeRef tname_DebuggerBrowsableAttribute, [typ_DebuggerBrowsableState], [ILAttribElem.Int32 n], [])
 
-  let mkDebuggerBrowsableNeverAttribute() =
-      match debuggerBrowsableNeverAttributeCache with
-      | None ->
-          let res = tref_DebuggerBrowsableAttribute 0
-          debuggerBrowsableNeverAttributeCache <- Some res
-          res
-      | Some res -> res
+  let debuggerBrowsableNeverAttribute = tref_DebuggerBrowsableAttribute 0
```

Comment:
I don't really see the point of this "cache". Even if the attribute never gets created during a compilation, it doesn't justify the mutable variable.